utils/mp-server/query-scripts/05-count-of-versions.sql.tsv

```diff
@@ -0,0 +1,17 @@
+CLIENT_VERSION	VERSION_COUNT
+1.14.11	2270
+1.14.9	672
+1.14.5	186
+1.14.7	125
+1.14.4	42
+1.14.6	29
+1.14.10	26
+1.14.9+dev	17
+1.14.3	17
+1.14.8	16
+1.14.1	7
+1.14.0	5
+1.14.2	4
+1.15.2	3
+1.14.10+dev	1
+1.14.11+dev	1
```

Comment:
I suppose this file was accidentally added?

Reply:
Yep.